draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -270,87 +213,64 @@ upstream HTTP server is out of scope for this document.
+an origin server while guaranteeing freshness and without the need for the
+server to transmit a nonce to the user agent. This allows the server to accept
 authenticated clients without revealing that it supports or expects
-authentication for some resources. It also allows authentication without
-the user agent leaking the presence of authentication to observers due to
+authentication for some resources. It also allows authentication without the
+user agent leaking the presence of authentication to observers due to
 clear-text TLS Client Hello extensions.
 
-The authentication proofs described in this document are not bound to individual
-HTTP requests; if the key is used for authentication proofs on multiple
-requests they will all be identical. This allows for better compression when
-sending over the wire, but implies that client implementations that multiplex
-different security contexts over a single HTTP connection need to ensure that
-those contexts cannot read each other's header fields. Otherwise, one context
-would be able to replay the unprompted authentication header field of another.
-This constraint is met by modern Web browsers. If an attacker were to compromise
-the browser such that it could access another context's memory, the attacker
-might also be able to access the corresponding key, so binding authentication to
-requests would not provide much benefit in practice.
-
-Key material used for authentication in unprompted authentication, whether
-symmetric or asymmetric MUST NOT be reused in other protocols. Doing so can
-undermine the security guarantees of the authentication.
-
-Origins offering Unprompted Authentication are able to link requests that use
-the same key for the Authentication Schemes provided. However, requests are
-not linkable across origins if the keys used are specific to the individual
-origins using Unprompted Authentication.
+The authentication proofs described in this document are not bound to
+individual HTTP requests; if the key is used for authentication proofs on
+multiple requests they will all be identical. This allows for better
+compression when sending over the wire, but implies that client implementations
+that multiplex different security contexts over a single HTTP connection need
+to ensure that those contexts cannot read each other's header fields.
+Otherwise, one context would be able to replay the Authorization header field
+of another. This constraint is met by modern Web browsers. If an attacker were
+to compromise the browser such that it could access another context's memory,
+the attacker might also be able to access the corresponding key, so binding
+authentication to requests would not provide much benefit in practice.
+
+Key material used for the Signature HTTP authentication scheme MUST NOT be
+reused in other protocols. Doing so can undermine the security guarantees of
+the authentication.
+
+Origins offering this scheme are able to link requests that use the same key.
```

Comment:
But these use the same connection anyway, so is this really a problem?  I guess if your goal is to break linkability, but only OHTTP does that (and this makes no real sense to do for the outer request).

Reply:
Not if you're using h1 though. I guess this is just a "sticking eye in fork hurts" sign, but it could be removed